references/bibtex/hh.bib

```diff
@@ -882931,3 +882931,23 @@ @book{s:Laribyan:Hayeren:1958
     macro_area = {Eurasia},
     title_english = {A new group of Armenian dialects}
 }
+
+
+@phdthesis{ahamb_rangelov,
+  author     = {Rangelov, Tihomir},
+  pages      = {iv+408},
+  school     = {University of Waikato},
+  title      = {A grammar of the Ahamb language (Vanuatu)},
+  year       = {2020},
+  hhtype     = {grammar},
+  inlg       = {English [eng]},
+  lgcode     = {Ahamb = [axam1237]},
+  url ={https://www.academia.edu/44669830/A_grammar_of_the_Ahamb_language_Vanuatu_},
```

Comment:
@HedvigS you should also change the URL to the one based on the DOI, I think, i.e. `https://doi.org/...`.

Reply:
Okay, sure. Whatever the standard is for Harald's bibs